packages/admin/cms-admin/src/generator/generateForm.ts

```diff
@@ -141,7 +141,7 @@ export async function writeCrudForm(generatorConfig: CrudGeneratorConfig, schema
     import { IconButton, FormControlLabel, MenuItem } from "@mui/material";
     import { FormApi } from "final-form";
     import isEqual from "lodash.isequal";
-    import React from "react";
```

Comment:
I also needed to update the generator code, because the imports in the generated files does not comply with the added linting rule. Should i do this in an own PR or is it ok to add it here?

Reply:
#2435